db/migrate/20191017175055_update_users_table.rb

```diff
@@ -1,7 +0,0 @@
-class UpdateUsersTable < ActiveRecord::Migration[5.2]
-  def change
-    add_column :users, :name, :string, null: false
```

Comment:
In general, you want to avoid deleting migrations, because it means that everyone on the team has to drop their databases, create, and migrate again. But I see that this info is all in the add_users_columns migration, so it's okay to delete as long as you message to your team that they'll need to recreate their dbs.

Reply:
it was a duplicate